whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+- Join the [Hugging Face Discord server](https://hf.co/join/discord) and make sure you have access to the #events channel.
+
+[comment]: # TODO: VB - create a post on the forum and link it above. Update the URL to cloud provider when decided.
+
+## Table of Contents
+
+- [TLDR;](#tldr)
+- [Important dates](#important-dates)
+- [How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries)
+- [Data and Preprocessing](#data-and-preprocessing)
+- [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model)
+- [How to fine-tune with OVH could](#how-to-finetune-with-ovh-cloud)
+- [How to combine n-gram language models with acoustic model](#how-to-combine-n-gram-with-acoustic-model)
+- [Evaluation](#evaluation)
+- [Prizes](#prizes)
+- [Communication and Problems](#communication-and-problems)
+- [Talks](#talks)
+- [General Tips & Tricks](#general-tips-and-tricks)
+
+## TLDR
+
+Participants are encouraged to leverage pre-trained speech recognition checkpoints,
+preferably [openai/whisper-large](https://huggingface.co/openai/whisper-large), 
+to train a speech recognition system in a language of their choice.
+
+Speech recognition systems should be trained using **PyTorch**, **🤗 Transformers**, and, **🤗 Datasets**.
+For more information on how to install the above libraries, please read through 
+[How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries).
+
+Participants can make use of whatever data they think is useful to build a 
+speech recognition system for **real-world** audio data - 
+**except** the Common Voice `"test"` split of their chosen language.
+The section [Data and preprocessing](#data-and-preprocessing) explains 
+in more detail what audio data can be used, how to find suitable audio data, and 
+how the audio data can be processed.
+
+For training, it is recommended to use the [official training script](https://github.com/huggingface/transformers/blob/main/examples/pytorch/speech-recognition/run_speech_recognition_seq2seq.py) or a modification thereof. A step-by-step guide on how to fine-tune 
+an acoustic model for a speech recognition system can be found under [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model).
+If possible it is encouraged to fine-tune the acoustic models on local GPU machines, but 
+if those are not available, the CLOUD_PROVIDER could team kindly provides a limited 
+number of GPUs for the event. Simply fill out [this google form](https://forms.gle/F2bpouvhDpKKisM39) to get access to a GPU.
+[comment]: # For more information on how to train an acoustic model on one of OVH's GPU - see [How to fine-tune a speech recognition model with OVHcould](#how-to-fine-tune-with-ovh-cloud).
+
+During the event, the speech recognition system will be evaluated on both the Common Voice `"test"` split 
+of the participants' chosen language as well as the *real-world* `"dev"` data provided by 
```

Comment:
Real-world dev was something specific to robust speech. I don't think this was discussed for Whisper?